playkit/src/main/java/com/kaltura/playkit/drm/WidevineModularAdapter.java

```diff
@@ -315,9 +316,9 @@ private MediaDrmSession openSessionWithKeys(FrameworkMediaDrm mediaDrm, String k
         return session;
     }
 
-    private byte[] executeKeyRequest(String licenseUrl, ExoMediaDrm.KeyRequest keyRequest) throws Exception {
+    private byte[] executeKeyRequest(String licenseUrl, ExoMediaDrm.KeyRequest keyRequest, String licenseServerUrl) throws Exception {
```

Comment:
Isn't `licenseUrl` (the first parameter to this function) the same as the one you just added (`licenseServerUrl`)?